VadeSecure/vade_secure_m365/_meta/fields.yml

```diff
@@ -0,0 +1,83 @@
+vadesecure.folder:
+    description: The name of the folder in the end user's mailbox if the message has been moved by Vade.
+    name: vadesecure.folder
+    short: vadesecure.folder
+    type: keyword
+vadesecure.from_header:
+    description: 'The email address declared in the From: header of the message, which may differ from the address used in the SMTP MAIL FROM command.'
+    name: vadesecure.from_header
+    short: vadesecure.from_header
+    type: keyword
+vadesecure.remediation_ids:
+    description: The remediation campaign associated with this message.
+    name: vadesecure.remediation_ids
+    short: vadesecure.remediation_ids
+    type: keyword
+vadesecure.status:
+    description: Filtering status
+    name: vadesecure.status
+    short: vadesecure.status
+    type: keyword
+vadesecure.substatus:
+    description: Filtering substatus
+    name: vadesecure.substatus
+    short: vadesecure.substatus
+    type: keyword
+vadesecure.to_header:
+    description: 'The email address declared in the To: header of the message, which may differ from the address used in the SMTP RCPT TO command.'
```

Comment:
```suggestion
    description: The email address declared in the To: header of the message, which may differ from the address used in the SMTP RCPT TO command.
```

Reply:
Quotes are needed as the value contains : character